test-plans/Makefile

```diff
@@ -0,0 +1,10 @@
+all: ping-image.tar
+
+ping-image.tar: Dockerfile Cargo.toml src
```

Comment:
Does this properly depend on all files in src? I thought you needed `wildcard`, but not sure.

Reply:
Yeah good point Marco, it seems to only depend on the directory creation/modification itself. Updated to use `wildcard`